Makefile

```
@@ -11,6 +11,7 @@
 .PHONY: install-rust install-rust-targets install-cbindgen
 .PHONY: ensure-probe
 .PHONY: vms/list vms/connect vms/release vms/init vms/probe vms/probe-rebuild vms/probe-report
+.PHONY: probe/validate
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)